src/chat.rs

```diff
@@ -219,6 +219,7 @@ impl ChatId {
                         }
                     }
                 }
+                Chattype::Mailinglist => bail!("Cannot protect mailing lists"),
```

Comment:
This is the sort of bugs catched with exhaustive matches.

Reply:
yes! +1